eng/Versions.props

```diff
@@ -174,7 +174,7 @@
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
     <GoogleProtobufPackageVersion>3.6.1</GoogleProtobufPackageVersion>
-    <GrpcAspNetCoreServerPackageVersion>0.1.19-pre1</GrpcAspNetCoreServerPackageVersion>
+    <GrpcAspNetCoreServerPackageVersion>0.1.19-pre2</GrpcAspNetCoreServerPackageVersion>
```

Comment:
This is an unlisted nightly build hosted on nuget.org

Reply:
Fine from my perspective. Check with @Eilon about approving the new version if you haven't already.

Reply:
@eilon, can you comment? Waiting for your green light before I merge.

Reply:
To be clear: this version update is to get the template successfully running with nightly builds of the .NET SDK.

Prior to preview 4 GA the template will be updated to a publicly list version of Grpc.AspNetCore.Server.